### ISSUE TITLE
How to run terminal command from external processes

### ISSUE THREAD

**Issue by brupelo** [microsoft/terminal#14524]
Here's the thing,

Right now in SublimeText I've been using a custom plugin that allows me to query the existing conemu profiles and pick one to open it directly in conemu, ie:

![image](https://user-images.githubusercontent.com/18058492/206875324-6563b7c3-f594-4bb1-a42a-45b0df573f31.png)

I'm considering to start using terminal instead conemu but I'd like to have a similar behaviour so I'd need to code a similar plugin. Questions:

- Where can I read the `terminal` profiles from? I guess they're stored in some json file somewhere?
- Let's assume I'm able to enumerate all existing `terminal` profiles from the file, how can I spawn some `terminal` commands to let's say split the pane & using the selected profile?

Basically the question boils down to know how I can spawn commands from an external process (client) to the terminal (server).

**Comment by brupelo** [comment]
@zadjii-msft Mike, thanks for the fast reply! Yeah, I think so... I haven't had the time yet to give it a shot yet but I think those will do pretty much... Once I've implemented this one as well as this other pending one [https://github.com/microsoft/terminal/issues/14312](https://github.com/microsoft/terminal/issues/14312) I think I'd be in a good spot to transition from conemu to this wonderful clean piece of software. As I said in other thread, you guys been doing some really kick-ass job on the new terminal, I love it :D
